runtime-modules/bounty/src/tests/mod.rs

```diff
@@ -4821,6 +4972,38 @@ fn switch_oracle_invalid_origin() {
     });
 }
 
+#[test]
+fn switch_oracle_fails_with_invalid_new_oracle_member_id() {
```

Comment:
Missing `switch_oracle_fails_with_invalid_member_controller_account` case

Reply:
You cannot test for `ensure_member_controller_account_origin` in this case. Since in `ensure_switch_oracle_actors` doesn't stop execution (no ?) 

If ensure_bounty_actor_manager returns an error, It will be `BadOrigin` from `ensure_root` or `MemberProfileNotFound` from `controller_account_id`
```rust
        let switcher_actor_manager =
            BountyActorManager::<T>::ensure_bounty_actor_manager(origin.clone(), current_oracle);

        let switcher = match switcher_actor_manager {
            Ok(oracle_switcher_manager) => oracle_switcher_manager.get_bounty_actor(),
            Err(_) => {
                ensure_root(origin)?;
                BountyActor::Council
            }
        };

        //Check if new oracle is a member
        BountyActorManager::<T>::get_bounty_actor_manager(new_oracle)?;
        Ok(switcher)
```

this is tested in `switch_oracle_fails_with_invalid_new_oracle_member_id`

Reply:
What I meant was to test the case where the caller is not a valid member controller of the **current** orcale, ie. this specific path:
`switch_oracle` => `ensure_switch_oracle_actors` => `ensure_bounty_actor_manager(origin.clone(), current_oracle)`

Reply:
[switch_oracle_fails_with_invalid_member_controller_of_current_oracle](https://github.com/Joystream/joystream/pull/3840/commits/d10800b7ab7f228b4a5a9bfe701e6af6e5f849a2)